# Data Processing Project - 

Finding Geolocations from Universities, FH (Fachhochschulen) are saved for later.
Data Source = https://www.data.gv.at/katalog/dataset/3e62e13e-3955-494a-91b3-f00ab9e84b48

Dict Unis - total 162 Datensätze
row[4] = id

1) GPS Koordinaten verkehrt - [y-, x-]
2) teilweise alte Gebäude!
3) Radius - 1.5 km?

- Car Sharing Parkplätze


In [91]:
import os
import csv
uniGPSFile = "./UNIVERSITAETOGD.csv"

with open(uniGPSFile) as f:
    uniloc = csv.reader(f, delimiter=",")
    next(uniloc)
    for row in uniloc:
        #id = row[0].strip().split('.')[1]
        print(row[4] + "\t" + row[2] + "\t" + row[3])

1546275	FH Campus Wien	Gebäude
1546268	Fachhochschule des bfi Wien	Hauptgebäude
1546269	Fachhochschule des bfi Wien	ENERGYbase
1546270	FHWien-Studiengänge der Wirtschaftskammer Wien	Gebäude
1546271	Lauder Business School	Gebäude
1546272	FH Technikum Wien	Geschäftsführung
1546273	FH Technikum Wien	Standort Höchstädtplatz
1546274	FH Technikum Wien	ENERGYbase
1545911	Universität Wien	Fakultät für Wirtschaftswissenschaften
1546397	Hochschule für Argar und Umweltpädagogik	Gebäude
1546276	Technische Universität Wien	Lehartrakt
1546277	Akademie der bildenden Künste Wien	Atelierhaus
1546289	Medizinische Universität Wien	Gebäude
1546290	Universität für Bodenkultur Wien	Zentrum für Internationale Beziehungen
1546291	Universität Wien	Hauptgebäude
1546292	Universität Wien	Universitätssportzentrum USZ II
1546294	Universität für Bodenkultur Wien	Baracken
1546295	Universität Wien	Theaterwissenschaft Hofburg
1546113	Universität für Musik und darstellende Kunst Wien	Palais Springer
1546114	Universität 

In [131]:
with open(uniGPSFile) as f:
    uniloc = csv.reader(f, delimiter=",")
    next(uniloc)
    for row in uniloc:
        if "1546148" in row[4]:
            #print(row[4]  + "\t" + row[2] + "\t" + row[3] + "\t\t" + gpsx + " " + gpsy)   
            print(row)

['UNIVERSITAETOGD.1546148', 'POINT (16.337577480698467 48.23654607282594)', 'Universität für Bodenkultur Wien', 'Gregor Mendel Haus (Hauptgebäude)', '1546148', '']


In [165]:
unigps = {}
with open(uniGPSFile) as f:
    uniloc = csv.reader(f, delimiter=",")
    next(uniloc) #skip header line
    for row in uniloc:
        gpsy = row[1].strip().split("(")[1].split(" ")[0]
        gpsx = row[1].strip().split("(")[1].split(" ")[1].split(")")[0]
        unigps[row[4]] = str(gpsx + "," + gpsy)

#unigps

{'1545911': '48.21930878706465,16.367516404807773',
 '1545922': '48.19878133969107,16.335922043292072',
 '1545923': '48.21545929381975,16.378428617289224',
 '1545924': '48.25701787424551,16.404303902211193',
 '1545925': '48.29413341666348,16.38007309851057',
 '1545926': '48.231370143001044,16.331424019483784',
 '1545927': '48.20800765788998,16.372900547103416',
 '1545928': '48.164824008924164,16.374147230618906',
 '1545929': '48.208851826766015,16.377203107870834',
 '1545930': '48.16678670437826,16.374117226157473',
 '1545931': '48.21384341856702,16.408860446502068',
 '1545932': '48.21274722310716,16.40990387836498',
 '1545933': '48.21220475172164,16.409839010138533',
 '1545934': '48.21396293277605,16.40641305840182',
 '1545935': '48.214569211541416,16.407354829589078',
 '1545936': '48.2146197497305,16.40626443464473',
 '1545937': '48.21566196605036,16.406223135786885',
 '1546113': '48.196989050609844,16.38231295437089',
 '1546114': '48.19658099664793,16.382282456047378',
 '1546115': '

## Datasets
1545931 - Wirtschaftsuniversität Wien', 'LC - Hauptgebäude',

1546291	Universität Wien	Hauptgebäude	 (16.360851928870705 48.213012828415096)

1546202	Technische Universität Wien	Hauptgebäude

1546148	Universität für Bodenkultur Wien	Gregor Mendel Haus (Hauptgebäude)

